docs/reference/contrib/frontendcache.rst

```diff
@@ -200,3 +204,62 @@ For example, this could be useful for purging a single page of blogs:
 
     # Purge the first page of the blog index
     purge_url_from_cache(blog_index.url + '?page=1')
+
+
+Invaliding many URLs in batches
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 1.13
+
+If you have many pages/URLs that need to be purged at the same time, it's more
+perfomant to use Wagtail's ``PurgeBatch`` class to build up a list of pages/URLs
+to purge, and purge them all in a single request.
+
+.. code-block:: python
+
+    # models.py
+    from django.dispatch import receiver
+    from django.db.models.signals import pre_delete
+
+    from wagtail.wagtailcore.signals import page_published
+    from wagtail.contrib.wagtailfrontendcache.utils import PurgeBatch
+
+
+    ...
+
+
+    def blog_page_changed(blog_page):
+        # Find all the live BlogIndexPages that contain this blog_page
+        batch = PurgeBatch()
+        for blog_index in BlogIndexPage.objects.live():
+            if blog_page in blog_index.get_blog_items().object_list:
+                batch.add_page(blog_index)
+
+        # Purge all the blog indexes we found in a single request
+        batch.purge()
+
+
+    @receiver(page_published, sender=BlogPage):
```

Comment:
Remove extra trailing `:`.

Reply:
@kaedroho the trailing `:` is still there